test/src/e2e.dynval/setup.ts

```diff
@@ -57,6 +57,7 @@ export function withNodes(
             result.map(node => node.testFailed(this.currentTest!.fullTitle()));
         }
         await Promise.all(result.map(node => node.clean()));
+        options.promiseExpect.checkFulfilled();
```

Comment:
`promiseExpect` is expected to be checked in `afterEach` of a test which it is declared.

Reply:
Ok, I added promiseExpect checking in `AfterEach` hook.